src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -839,59 +709,62 @@ private void DrawPackages(IEnumerable<INugetPackage> packages, bool canBeSelecte
         /// </summary>
         private void DrawOnlineHeader()
         {
-            var headerStyle = GetHeaderStyle();
-
-            EditorGUILayout.BeginVertical(headerStyle);
+            EditorGUILayout.BeginVertical(Styles.BackgroundStyle);
             {
-                EditorGUILayout.BeginHorizontal();
+                EditorGUILayout.BeginHorizontal(EditorStyles.toolbar);
                 {
-                    var showPrereleaseTemp = EditorGUILayout.Toggle("Show Prerelease", showOnlinePrerelease);
+                    var showPrereleaseTemp = GUILayout.Toggle(
+                        showOnlinePrerelease,
+                        showPrereleaseContent,
+                        EditorStyles.toolbarButton,
+                        GUILayout.Width(120f));
                     if (showPrereleaseTemp != showOnlinePrerelease)
                     {
                         showOnlinePrerelease = showPrereleaseTemp;
                         UpdateOnlinePackages();
                     }
 
+                    GUILayout.FlexibleSpace();
                     DrawSelectFromClipboardButton();
                     DrawMandatoryButtons();
                 }
 
                 EditorGUILayout.EndHorizontal();
 
-                var enterPressed = Event.current.Equals(Event.KeyboardEvent("return"));
-
-                EditorGUILayout.BeginHorizontal();
+                var style = new GUIStyle(EditorStyles.toolbar) { fixedHeight = 25f };
+                EditorGUILayout.BeginHorizontal(style);
                 {
-                    var oldFontSize = GUI.skin.textField.fontSize;
-                    GUI.skin.textField.fontSize = 25;
-                    onlineSearchTerm = EditorGUILayout.TextField(onlineSearchTerm, GUILayout.Height(30));
+                    var enterPressed = Event.current.Equals(Event.KeyboardEvent("return"));
 
-                    if (GUILayout.Button("Search", GUILayout.Width(100), GUILayout.Height(28)))
+                    // draw search field
+                    onlineSearchTerm = EditorGUILayout.TextField(onlineSearchTerm, Styles.SearchFieldStyle, GUILayout.Height(20));
+
+                    if (GUILayout.Button("Search", GUILayout.Width(100), GUILayout.Height(20)))
                     {
                         // the search button emulates the Enter key
                         enterPressed = true;
                     }
 
-                    GUI.skin.textField.fontSize = oldFontSize;
+                    // search only if the enter key is pressed
+                    if (enterPressed)
+                    {
+                        GUI.FocusControl(string.Empty);
```

Comment:
Why do you deselect the search box? It seems it would be useful to modify the search term if you search didn't provide satisfactory results so you can try again.

Reply:
This is a piece of code I wrote when I tried to remove the Search button, but it ended up being unnecessary as I kept the button. Therefore, I removed it.